src/tools/ci/docker/linux_opensuse_tumbleweed

```diff
@@ -22,12 +22,14 @@ RUN \
   gcc \
   gcc-c++ \
   gettext-runtime \
+  lz4 \
   ninja \
   \
   libicu-devel \
   libldapcpp-devel \
   liblz4-devel \
   libopenssl-devel \
+  libselinux-devel \
   libuuid-devel \
```

Comment:
Doesn't matter sufficiently here, but in future it's better to do stuff like this separately.